### PR TITLE
To do day1

### PR DESCRIPTION
**Pull request by basem909**
## Description
- This project aims to create a simple to-do list on daily basis 
- The requirements for this part:
- Project requirements
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
## Technologeis and plugins
Webpack
Webpack CLI
Html Webpack Plugin
Webpack Dev Server
Style Loader
CSS Loader

### CODE REVIEW

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @basem909 , 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is difficult to review because of the present of node_module in your repository 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @basem909  👋,

_This is an additional review as a result of the second opinion request._

Great work on deleting the node_modules from Github. Remember this folder and its contents are always generated based on the environment and versions of dependencies available and compatible. A simple `npm i` does this magic. For this reason, its not a best practice to push this folder since it varies from environment/computer to time.

It is clear that you deleted this file 8 hours or so ago but it was pushed to Github in your first commit, For this reason, the reviewer has more than 5000 files for him to make a review which overwhelms any reviewer and thus a thorough review can't be given.

> NOTE: This review request was invalidated so you still have 3 review attempts left for this project. If they are less than 3, kindly let me know so that I can restore this, or better yet, review your project myself. 😄 

Submit another CRR whenever you are ready.

Keep rocking 🚀!

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```
Comment (medaminedev66) [inline webpack.config.js]: - [ ] The 'mode' option has not been set in the `webpack.confog.js`, Webpack will fallback to 'production' for this value.
I recommend setting the 'mode' option to 'development'or 'production' to enable defaults for each environment.
Learn more: https://webpack.js.org/configuration/mode/

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @basem909   😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️
- The Webpack is used to set up the project. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @basem909, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] Kindly remove `node_modules` from the repo and include it in your `.gitignore` file to make the repo more cleaner and lightweight

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**